Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFloor.cs

```diff
@@ -280,9 +265,43 @@ private ICurve GetFlattenedCurve(ICurve curve, double z)
             return newArcCurve;
           }
 
-        //case OG.Circle circle:
-        //case OG.Ellipse ellipse:
-        //case OG.Spiral spiral:
+        // Note: this method is untested. It seems Revit doesn't send circles... it sends two arcs instead.
```

Comment:
Unfortunately, known limitation of Revit, I think we had to deal with the same for walls